packages/core/platform/Makefile

```
@@ -1,23 +1,20 @@
 NAME=platform
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)